_posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Adopting Docs-as-Code: From Hackathon to Production"
+excerpt: Sysdig's journey from unstructured to structured, finally to semi-structured authoring and how Sysdig hackathon enabled design their homegrown docs-like-code solution.
+last_modified_at: 
```

Comment:
You can enter `last_modified_at: Mon Jul 11 17:33:53 CDT 2022` to match the format of other posts.